.changeset/eleven-doors-shop.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I think we can do this without releasing a new version right? Or do we need to release a new one because the previous one might be broken?

Reply:
I think it would have failed publishing if it was broken, so yeah, we don't need to version